system/ns/enter.go

```diff
@@ -52,15 +46,12 @@ func Enter(ns netns.NsHandle) (func() error, error) {
 // NsCreate returns a handle to a new network namespace.
 // NsEnter must be used to safely enter and exit the new namespace.
 func Create() (netns.NsHandle, error) {
-	runtime.LockOSThread()
-	defer runtime.UnlockOSThread()
-
 	origns, err := netns.Get()
 	if err != nil {
 		return netns.None(), err
 	}
 	defer origns.Close()
 	defer netns.Set(origns)
 
-	return netns.New()
+	return netns.NewNamed("kola-test")
```

Comment:
This creates a single global namespace, or? Then we couldn't run many kola instances in parallel.

Do you think there is a way to keep the functions as they are and try to rather change where the new network namespace gets created and pass its handle along so that `.Enter()` can be used?

Reply:
It was for testing purposes - using named namespace, it's easier to `nsenter`.

Reply:
Ah, ok, then without it one has to use the PID: `nsenter --net -t PID`